### PR TITLE
[RORDEV-1257] ECK+ROR tested using E2E tests 

### PR DESCRIPTION
**Pull request by coutoPL**
* new environment: ECK

### CODE REVIEW

File: environments/eck-ror/readme.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# README
+
+## Requirements:
```
Comment (Dzuming) [inline environments/eck-ror/readme.md:3]: is there a minimum version for these required tools?

File: environments/eck-ror/readme.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+* docker installed
+* kind tool installed (https://github.com/kubernetes-sigs/kind)
+
+## RUNNING
+1. Running the ECK+ROR PoC: `$ ./start.sh --es <ES_VESION> --kbn <KBN_VERSION>`
```
Comment (Dzuming) [inline environments/eck-ror/readme.md:8]: Is it still PoC?

File: environments/elk-ror/docker-compose.yml [hunk 3]
```diff
@@ -22,6 +22,9 @@ services:
       - bootstrap.memory_lock=true
       # for a sake of debugging and profiling
       - "ES_JAVA_OPTS=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5000 -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=9010 -Dcom.sun.management.jmxremote.local.only=false -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.rmi.port=9010 -Djava.rmi.server.hostname=127.0.0.1"
+      - KIBANA_SERVICE_ACCOUNT_TOKEN=UNUSED
```
Comment (Dzuming) [inline environments/elk-ror/docker-compose.yml:25]: Must we pass these dummy ECK-based values?

**Review by Dzuming — APPROVED** [review]
LGTM  I left few small non-blocking comments

File: environments/eck-ror/readme.md (hunk 2 above)
Comment (coutoPL) [reply environments/eck-ror/readme.md]: I removed the file. We don't need it.

File: environments/elk-ror/docker-compose.yml (hunk 3 above)
Comment (coutoPL) [reply environments/elk-ror/docker-compose.yml:26]: For now - yes. ROR settings are a little bit different in the case of ECK (we have to add two more blocks: for the probe and for a sake of the ECK operator... and we cannot use /at least in a simple way/ kibana user:pass block).
These ENVs are used in these additional blocks. Currently, ROR ES doesn't support fallback to default value. When ENV is not found, ROR settings fail to reload. That's why we have to add the dummy values here.

**Review by Dzuming — APPROVED** [review]
LGTM!

**Pull request merged**